ext/MitsubaLoader/CElementShape.cpp

```diff
@@ -171,6 +171,7 @@ bool CElementShape::addProperty(SNamedPropertyElement&& _property)
 	auto setCollapse = SET_PROPERTY_TEMPLATE(collapse,SNamedPropertyElement::Type::BOOLEAN, Obj);
 	auto setSRGB = SET_PROPERTY_TEMPLATE(srgb,SNamedPropertyElement::Type::BOOLEAN, Ply);
 	auto setShapeIndex = SET_PROPERTY_TEMPLATE(shapeIndex,SNamedPropertyElement::Type::INTEGER, Serialized);
+	//auto setToWorld = SET_PROPERTY_TEMPLATE(toWorld, SNamedPropertyElement::Type::MATRIX, Instance);
```

Comment:
what happened here!?

Reply:
nothing, code wrote by me and commented-out by me. Shouldnt be here